external/buildscripts/Build.bee.cs

```diff
@@ -124,10 +119,10 @@ private static void RegisterCommonNonWindowsArtifacts()
 					"autoconf-src/2.69_0e4ba7a0363c68ad08a7d138b228596aecdaea68e1d8b8eefc645e6ac8fc85c7.7z",
 					"public"));
 
-			Artifacts.Add("libgdiplus",
+			Artifacts.Add("libgdiplus-mac",
```

Comment:
I am not sure where this is used. I could not find this path being used in build.pl.

Reply:
I think we used to copy it into the build.7z. Perhaps we don't bother anynmore?